lib/util.js

```diff
@@ -615,8 +615,8 @@ function formatValue(ctx, value, recurseTimes, ln) {
 }
 
 function formatNumber(fn, value) {
-  // Format -0 as '-0'. Strict equality won't distinguish 0 from -0.
-  if (Object.is(value, -0))
+  // Format -0 as '-0'. A `value === -0` check won't distinguish 0 from -0.
+  if (1 / value === -Infinity)
```

Comment:
Might be worth benchmarking `1 / value < 0` also.

Reply:
@ljharb doesn't that have a different meaning? That condition is true if `value` is `-Infinity` but we want to know if `value` is `-0`.

Reply:
Actually `1 / -Infinity` is `-0`, so a `< 0` check would return false for that `value = -Infinity`. I think having the stricter check is better.

Reply:
Ah yes, good call. Don't mind me :-)